sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample01_DataFeedCrudOperations.cs

```diff
@@ -0,0 +1,184 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.AI.MetricsAdvisor.Administration;
+using Azure.AI.MetricsAdvisor.Models;
+using Azure.AI.MetricsAdvisor.Tests;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.MetricsAdvisor.Samples
+{
+    [LiveOnly]
+    public partial class MetricsAdvisorSamples : MetricsAdvisorTestEnvironment
+    {
+        [RecordedTest]
```

Comment:
nit: It shouldn't make any difference, but for clarity this should be a `[Test]` attribute if it is going to be attributed as `[LiveOnly]`